src/main/java/org/opensearch/security/securityconf/ConfigModelV6.java

```diff
@@ -562,6 +566,10 @@ public boolean isPermittedOnSystemIndex(String indexName) {
             }
             return isPatternMatched && isPermitted;
         }
+
+        private boolean isSystemIndex(String indexName, WildcardMatcher systemIndexMatcher) {
```

Comment:
this check should be moved to SecurityIndexSearcherWrapper.java

Reply:
okay